### PR TITLE
Show all partners on tools, process partners on call tool backend

### PR DESCRIPTION
**Pull request by vbrown608**
* Show all partners on tools and process all tools on the backend. This is a step towards randomly selecting a partner on the frontend - I wanted to render and process them all correctly so this will work for no-js users.

* Process partner submissions for the call tool.

* Stop using a special param to do determine if we should show the EFF email signup. We can handle this using the partner param in #314.

* Add an end-to-end integration test for congress forms, using the puffing billy gem to mock ajax calls to the contact congress service.

### CODE REVIEW

**Comment by wioux** [comment]
The styling on the call tool radios is off. There is the an issue with that on production as well (#322), but perhaps we could fix it in this PR?

File: app/views/tools/_call.html.erb [hunk 1]
```diff
@@ -119,10 +119,8 @@
           </div>
 
           <p style="text-align: center;">
-            <% if ask_for_newsletter_signup? %>
             <br />
             <strong><a href="/register">Sign up</a> for an EFF Action account and save your information for next time.</strong>
```
Comment (wioux) [inline app/views/tools/_call.html.erb]: It still makes sense to hide this for logged-in users.

File: features/step_definitions/congress_steps.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Given(/^a congress member exists$/) do
+  member = FactoryGirl.create(:congress_member)
+end
+
+Given(/^a stubbed phantomdc$/) do
+  Rails.application.config.congress_forms_url = "http://phantomdc.test"
```
Comment (wioux) [inline features/step_definitions/congress_steps.rb]: Awesome! Though it further confuses me about how exactly cucumber/capybara works...

**Comment by wioux** [comment]
This looks pretty good to me. I made a few notes but nothing is really blocking.  :+1:

File: app/views/tools/_call.html.erb (hunk 1 above)
Comment (vbrown608) [reply app/views/tools/_call.html.erb]: Good point

**Pull request merged**